DRAFT_RELEASE_NOTES.md

```diff
@@ -1,8 +1,9 @@
 # Major Additions
```

Comment:
S: Need to add this migration name and description to the baton migration table in the `Automatic Upgrades` section of the release notes.

Reply:
@carter-cundiff update! thanks for catching